src/more-infos/more-info-group.js

```diff
@@ -36,4 +39,40 @@ export default new Polymer({
       ],
     },
   },
+
+  observers: [
+    'statesChanged(stateObj, states)',
+  ],
+
+  statesChanged(stateObj, states) {
+    let groupDomainStateObj = false;
+
+    if (states && states.length > 0) {
+      const baseStateObj = maxBy(states, (s) => Object.keys(s.attributes).length);
+
+      groupDomainStateObj = baseStateObj.set('entityId', stateObj.entityId).set(
+          'attributes', Object.assign({}, baseStateObj.attributes));
+    }
+
+    for (const s of states) {
```

Comment:
I'm not sure which line you are referring to here. Won't babel take care of any browser normalization when it is built anyway?